docs/modules/ROOT/pages/usage.adoc

```diff
@@ -116,3 +135,44 @@ The operator creates a `ConfigMap` with the name of the cluster which contains c
 - `DRUID_ROUTER` with the format `<host>:<port>`, which points to the router processes HTTP endpoint. Here you can connect to the web UI, or use REST endpoints such as `/druid/v2/sql/` to query data. https://druid.apache.org/docs/latest/querying/sql.html#http-post[More information in the Druid Docs].
 - `DRUID_AVATICA_JDBC` contains a JDBC connect string which can be used together with the https://calcite.apache.org/avatica/downloads/[Avatica JDBC Driver] to connect to Druid and query data. https://druid.apache.org/docs/latest/querying/sql.html#jdbc[More information in the Druid Docs].
 - `DRUID_SQALCHEMY` contains a connection string used to connect to Druid with SQAlchemy, in - for example - Apache Superset.
+
+== Configuration Overrides
+
+The cluster definition also supports overriding specific configuration properties.
+
+IMPORTANT: Overriding certain properties which are set by  operator (such as the HTTP port) can interfere with the operator and can lead to problems.
+
+For a role or rolegroup, at the same level of `config`, you can specify: `configOverrides`, for the `runtime.properties`. For example, if you want to set the `druid.server.http.numThreads` for the router to 100 adapt the `routers` section of the cluster resource like so:
+
+[source,yaml]
+----
+routers:
+  roleGroups:
+    default:
+      config: {}
+      configOverrides:
+        runtime.properties:
+          druid.server.http.numThreads: "100"
+      replicas: 1
+----
+
+Just as for the `config`, it is possible to specify this at role level as well:
+
+[source,yaml]
+----
+routers:
+  configOverrides:
+    runtime.properties:
+      druid.server.http.numThreads: "100"
+  roleGroups:
+    default:
+      config: {}
+      replicas: 1
+----
+
+All override property values must be strings.
+
+For a full list of configuration options we refer to the Druid https://druid.apache.org/docs/latest/configuration/index.html[Configuration Reference].
```

Comment:
I would add a warning / attention field where we refer to ports. You can override ports, but since we hardcode that (container / service ports) it will probably break the whole thing?

Reply:
there is a warning at the beginning of the section

Reply:
Maybe that would be the one usecase where we actually should read the ports from the config ;) ? not sure if it is worth though.

Reply:
The warning also mentions the HTTP port

Reply:
yeah did not read that :D